website/docs/reference/resource-configs/clickhouse-configs.md

```diff
@@ -186,11 +188,32 @@ dbt snapshots allow a record to be made of changes to a mutable model over time.
 
 </File>
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+<File name='snapshots/<model_name>.sql'>
+
+```jinja
+{{
+   config(
+     schema = "<schema_name>",
+     unique_key = "<column-name>",
+     strategy = "<strategy>",
+     updated_at = "<unpdated_at_column-name>",
+   )
+}}
+```
+
+</File>
+
+</VersionBlock>
+
 #### Snapshot Configuration
 
 | Option          | Description                                                                                                               | Required?                                                                            |
 |-----------------|---------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
-| `target_schema` | A ClickHouse's database name where the snapshot table will be created.                                                    | Required                                                                             |
+| <VersionBlock lastVersion="1.8"> `target_schema` </VersionBlock> <VersionBlock firstVersion="1.9"> `schema` </VersionBlock>| A ClickHouse's database name where the snapshot table will be created.       | <VersionBlock lastVersion="1.8"> Required </VersionBlock>  <VersionBlock firstVersion="1.9"> Optional </VersionBlock>          |
 | `unique_key`    | A tuple of column names that uniquely identify rows.                                                                      | Required. If not provided altered rows will be added twice to the incremental table. |
```

Comment:
Incremental table? This is about snapshots not incremental models..

Reply:
Good catch! It looks like this page could use some larger love. I'll make a note to go back and review a lot of the community adapter pages